dbus_lynx_distributor/ftdi.py

```diff
@@ -1,186 +1,51 @@
+from pyftdi import FtdiLogger                                                             
+from pyftdi.ftdi import Ftdi                                                              
+from pyftdi.i2c import I2cController, I2cNackError                                        
+from pyftdi.misc import add_custom_devices   
+from pyftdi.ftdi import Ftdi as PyFtdi                                             
+import logging
 from time import time
 import usb
 
-
 class Ftdi:
```

Comment:
Unsure if we really need this class at all or we could do the fetid calls directly in the other file, but fine for now.

Reply:
with the low amount of code utilizing pyftdi likely it could be merged